src/utils/sendPaginatedList.js

```diff
@@ -15,11 +16,20 @@ async function sendPaginatedList(res, Model, filter, pageNo, limit) {
     if (limit * pageNo > filteredProductCount) {
```

Comment:
Let say we have a call, where `limit = 10` and `pageNo = 2`, and our `filteredProductCount is 14`. 20 is greater than 14, you will give error message, but you must give user the second page, which is list, where length is 4.
`This is wrong validation. `
I don't know why you deleted validation you had before. You must count `pageCount`, and check if `pageNo > pageCount`, and if yes, then give user 400, and if no, give the response